src/applications/appeals/10182/tests/utils/upload.unit.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
+
+import { focusFileCard, evidenceUploadUI } from '../../utils/upload';
+
+describe('evidenceUploadUI', () => {
+  describe('focusFileCard', () => {
+    const { container } = render(
+      <div>
+        <ul className="schemaform-file-list">
+          <li>test-file.png</li>
+          <li>test-file.pdf</li>
+        </ul>
+      </div>,
+    );
+
+    focusFileCard('test-file.pdf', container);
+    expect(document.activeElement).to.eq($('li:last-child', container));
+  });
+
+  describe('parseResponse', () => {
+    it('should return undefined', () => {
```

Comment:
does this return `undefined` for a particular value? seems like it's returning an object with some file data in it

Reply:
It looks like I mis-labeled that unit test!